pkg/provisioning/service.go

```diff
@@ -418,6 +461,76 @@ func (s *Service) createProcessors(ctx context.Context, r *rollback.R, parentID
 	return nil
 }
 
+func (s *Service) deleteOldPipelines(ctx context.Context, pipelines []string) {
+	var err error
```

Comment:
Nitpick: this line can be removed if we use `:=` when assigning to the variable.